Manifests/ManagedPreferencesApplications/com.trusourcelabs.NoMAD.plist

```diff
@@ -1434,7 +1434,7 @@ Line two</string>
 			<key>pfm_type</key>
 			<string>string</string>
 			<key>pfm_value_placeholder</key>
-			<string>Please provide your local password.</string>
+			<string>Your network and local passwords are not the same. Please enter the password for your Mac.</string>
```

Comment:
Isn't the assumption about MessageLocalSync that the *local password* does not match the network password and therefore they need to enter their *network* password to resync with the local account?

If that's the case then to me the wording for the placeholder should be different.

Reply:
If I had only read the documentation ...